django/src/rdwatch/models/site_evaluation.py

```diff
@@ -15,6 +16,7 @@
 
 
 class SiteEvaluation(models.Model):
+    created = CreationDateTimeField()
```

Comment:
Is the timestamp field technically a created time already?
https://github.com/ResonantGeoData/RD-WATCH/blob/main/django/src/rdwatch/models/site_evaluation.py#L76

Reply:
Good catch. I changed it to use that instead and backed out the `created` change.